server/remote_asset/fetch_server/fetch_server.go

```diff
@@ -137,7 +137,7 @@ func (p *FetchServer) FetchBlob(ctx context.Context, req *rapb.FetchBlobRequest)
 				log.CtxInfof(ctx, "FetchServer failed to get metadata for %s: %s", expectedSHA256, err)
 				continue
 			}
-			blobDigest.SizeBytes = md.SizeBytes
+			blobDigest.SizeBytes = md.DigestSizeBytes
```

Comment:
I think there's a rollout issue here because DigestSizeBytes may not be populated until all the apps are updated with the other changes. You can fall back to the old field if the new one is not set.